server/src/main/java/ru/practicum/shareit/item/model/Item.java

```diff
@@ -50,7 +50,7 @@ public class Item {
     @NotNull
     private User owner;
 
-    @ManyToOne(fetch = FetchType.LAZY)
-    @JoinColumn(name = "request_id", nullable = false)
+    @ManyToOne(fetch = FetchType.EAGER)
```

Comment:
всегда лучше делать FetchType.LAZY

Reply:
с LAZY не работает. Выдается ошибка `"com.fasterxml.jackson.databind.exc.InvalidDefinitionException: No serializer found for class org.hibernate.proxy.pojo.bytebuddy.ByteBuddyInterceptor and no properties discovered to create BeanSerializer (to avoid exception, disable SerializationFeature.FAIL_ON_EMPTY_BEANS) "`
Я так и не смола с ней разобраться

Reply:
попробуй все таки с этим разобраться: что бы игнорировать прокси-объекты по умолчанию, попробуй использовать аннотацию @JsonIgnoreProperties